src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Sort all persons in the address book and list to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book sorted by names.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
+    public CommandResult execute() {
+        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+        List<ReadOnlyPerson> sortedPersons = new ArrayList<ReadOnlyPerson>();
+        int i,j;
+        for(i=0;i<allPersons.size();i++){
+            sortedPersons.add(allPersons.get(i));
+        }
+        int n=allPersons.size();
+        for(i=0;i<n-1;i++){
```

Comment:
Instead of doing this. Can you think of a better way? Think about what attribute your sorting is based on. If you are not sure. You can refer to your peers's code for reference.